src/less/pricingEstimate.less

```diff
@@ -55,6 +55,7 @@
       color: @gray-300;
     }
   }
+
 }
 
 #createFormContainer {
```

Comment:
We really make people use the ID for a price estimator? Shouldn't this be a class?